Makefile

```diff
@@ -205,7 +205,19 @@ test-e2e: e2e-setup
 			--up-local \
 			--go-test-flags "-timeout=15m" \
 			--local-operator-flags "$(ZAP_FLAGS)" \
-			$(OPERATOR_SDK_EXTRA_ARGS)
+			$(OPERATOR_SDK_EXTRA_ARGS) \
+			| tee test-e2e.log
```

Comment:
@pmacik could we store those logs in the `out` folder? that's already in `.gitignore`, and when executing `make clean`, logs would be cleaned up as well.